stack.yaml

```diff
@@ -60,8 +60,8 @@ packages:
   extra-dep: true
 
 - location:
-    git: https://github.com/input-output-hk/cardano-crypto
-    commit: e87fcd3fea18f664d6e4f39c11abb8cafbd25785
+    git: https://github.com/angerman/cardano-crypto
```

Comment:
Do you not have access to this repo?
Can we put a TODO in there to explain why the fork and reminder to move it back to input-output-hk.

Reply:
I'll leave https://github.com/input-output-hk/cardano-crypto/pull/46 just here. But add it to the stack.yaml as well.

Reply:
Cool, thanks.